README.md

```diff
@@ -6,7 +6,8 @@
 
 An implementation of [`IAsyncEnumerable<'T>`][3] as a computation expression: `taskSeq { ... }` with an accompanying `TaskSeq` module, that allows seamless use of asynchronous sequences similar to F#'s native `seq` and `task` CE's.
 
-Latest version [can be installed from Nuget][nuget]. 
+* Latest stable version: [0.3.0 is on NuGet][nuget].
```

Comment:
perhaps the 2 version badges a la https://github.com/jet/equinox#readme might be more maintainable?